pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1673,10 +1677,18 @@ func handleModify(ctx *domainContext, key string,
 			status.ClearError()
 			publishDomainStatus(ctx, status)
 			doInactivate(ctx, status, false)
+			if status.HasError() {
+				log.Errorf("doInactivate failed: Aborting the modify of %s", status.DisplayName)
+				return
+			}
 			updateStatusFromConfig(status, *config)
 			changed = true
 		} else if status.Activated {
 			doInactivate(ctx, status, false)
```

Comment:
Can we have an existing errror here which needs to be cleared before inactivating?

Reply:
If there is an error then it will go in the upper if statement.